src/openzaak/components/catalogi/api/validators.py

```diff
@@ -443,3 +445,48 @@ def __call__(self, attrs):
                             f"Objects can't be updated with a relation to non-concept {field_name}"
                         )
                         raise ValidationError(msg, code=self.code)
+
+
+class GeldigheidVersiedatumValidator:
```

Comment:
@joeribekker @sergei-maertens @annashamray if the `versiedatum` is read-only in the admin, shouldn't it also be read-only in the API?

Reply:
@stevenbal Good observation but the answer is no (for now).

We create a UI where we make assumptions about how to fill the `versiedatum`. If this is the defacto standard way of doing things, then I guess it will be changed in the API spec but for now it's fine.

Reply:
only if it's properly described how the versioning machinery works, which isn't the case currently. if the 1.0.0 API spec says writable, then we have to stick to that :thinking: